readme.md

```diff
@@ -183,7 +187,7 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
 
 * **Question:** The above can be described as middleware. Describe what middleware is in your own words and how it differs from building a route.
 
-* **Your Answer:**
+* **Your Answer:** Middleware is the code that runs before any route is met. Middleware can be used to do run authentication, validation or test.
```

Comment:
Correct, although Middleware can also be applied between routes as well as before them.